lib/util.js

```diff
@@ -431,8 +431,8 @@ exports.nextSleepTime = function (
  * @returns {JSON} return next sleep Time and totalTime.
  */
 exports.jitteredSleepTime = function (numofRetries, currentSleepTime, totalTimeout, maxLoginTimeout) {
```

Comment:
better to change the argument name to maxRetryTimeout to avoid confusing.

Reply:
Replaced.